packages/account-abstraction/src/SmartAccountAPI.ts

```diff
@@ -159,12 +159,6 @@ export class SmartAccountAPI extends BaseAccountAPI {
       }
     }
 
-    if (!maxFeePerGas || !maxPriorityFeePerGas) {
```

Comment:
why do we need to remove this?

Reply:
there is no need. these variables are already assigned